### PR TITLE
BankingStage Refactor: Committer Simplify

### PR DESCRIPTION
**Pull request by apfitzge**
#### Problem
Some clean up in `committer.rs`.

#### Summary of Changes

Fixes #

### CODE REVIEW

File: core/src/banking_stage/committer.rs [hunk 1]
```diff
@@ -34,7 +34,6 @@ impl Committer {
         batch: &TransactionBatch,
         loaded_transactions: &mut [TransactionLoadResult],
         execution_results: Vec<TransactionExecutionResult>,
-        sanitized_txs: &[SanitizedTransaction],
```
Comment (taozhu-chicago) [inline core/src/banking_stage/committer.rs:37]: Good to have lesser arguments

**Review by taozhu-chicago — APPROVED** [review]
lgtm,

File: core/src/banking_stage/committer.rs (hunk 1 above)
Comment (apfitzge) [reply core/src/banking_stage/committer.rs:37]: ^ yeah, already passing the batch which contains the sanitized transactions

File: core/src/banking_stage/committer.rs [hunk 2]
```diff
@@ -88,25 +82,22 @@ impl Committer {
...
-                    bank,
-                    batch,
-                    pre_balance_info,
-                    starting_transaction_index,
-                );
-            },
-            "find_and_send_votes",
-        );
-        execute_and_commit_timings.find_and_send_votes_us = find_and_send_votes_time.as_us();
+        let ((), find_and_send_votes_us) = measure_us!({
```
Comment (ryoqun) [inline core/src/banking_stage/committer.rs:85]: :heart: for `()`

**Pull request merged**